AudioExample/AudioExample/Controllers/EqualizerViewModel.swift

```diff
@@ -43,7 +43,7 @@ final class EqualzerViewModel {
         if enable {
             equalizerService.activate()
         } else {
-            equalizerService.deactive()
```

Comment:
I guess this can be renamed to `deactivate`

Reply:
Fixed